test/test_mps.py

```diff
@@ -38,13 +38,14 @@
     SpectralFuncInfo,
     BinaryUfuncInfo,
 )
-from torch.testing._internal.common_device_type import ops, dtypes, instantiate_device_type_tests
+from torch.testing._internal.common_device_type import ops, dtypes, instantiate_device_type_tests, OpDTypes
 from torch.testing._internal.common_nn import NNTestCase
 import numpy as np
 import torch
 import torch.utils._pytree as pytree
 from itertools import product
 
+test_consistency_op_db = copy.deepcopy(op_db)
```

Comment:
op_db for TestConsistency will be in-place modified, deep-copy one for it to prevent other tests from being affected.